.github/workflows/run-scenarios.yaml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
 jobs:
   run-scenarios:
+    strategy:
+      matrix:
+        bases: [ development, mainnet, goerli, kovan, fuji ]
```

Comment:
Nice, I think we could even drop kovan now?

Reply:
Sure, I'll remove it.

Reply:
Done